src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/imageBase64.js

```diff
@@ -0,0 +1,46 @@
+class ImageBase64Visualization extends Visualization {
+    static inputType = 'Any'
+    static label = 'Image'
+
+    onDataReceived(data) {
+        function image(mediaType, base64Data) {
+            return (
+                '<img style="max-height:100%; max-width:100%;" src="data:' +
+                mediaType +
+                ';base64,' +
+                base64Data +
+                '" />'
+            )
+        }
+
+        while (this.dom.firstChild) {
+            this.dom.removeChild(this.dom.lastChild)
+        }
+
+        let parsedData = data
+        if (typeof data === 'string') {
+            parsedData = JSON.parse(data)
+        }
+        const mediaType = parsedData.mediaType || 'image/png'
+        const base64 = parsedData.base64
+
+        const width = this.dom.getAttributeNS(null, 'width')
+        const height = this.dom.getAttributeNS(null, 'height')
+        const tabElem = document.createElement('div')
+        tabElem.setAttributeNS(null, 'id', 'vis-image-view')
+        tabElem.setAttributeNS(null, 'class', 'scrollable')
```

Comment:
I don't think that it should use `scrollable` class, rather `visualization`. Scrollable only adds scrollbars if it overflows, but visualization adds rounded corners and i think drop shadow too?

Reply:
Sure, I added it by analogy with other visualizations. Removed :+1: